conda-recipes/faiss-gpu-feedstock/recipe/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+ARCH=`uname -p`
+if [[ "${ARCH}" == 'x86_64' ]]; then
+    # Build avx2 version
+    CXXFLAGS="-mavx2 -mf16c" ./configure
+    make -j $CPU_COUNT
+    make -C python _swigfaiss_avx2.so
+    make clean
+fi
+
+# Build vanilla version (no avx)
```

Comment:
wouldn't this section go into an else statement for the if statement above?

Reply:
The python code makes an explicit check of the CPU capabilities at runtime and then loads either the AVX2 or "default" flavor of the library (and will fall back to default if AVX2 load fails). So the idea is to build both libraries if possible, and leave the option to the runtime.